test/plugin/scenarios/jetty-scenario/jettyclient-scenario/src/main/java/org/apache/skywalking/apm/testcase/jettyclient/controller/CaseController.java

```diff
@@ -45,6 +46,8 @@ public void init() throws Exception {
     @ResponseBody
     public String jettyClientScenario() throws Exception {
         client.newRequest("http://" + jettyServerHost + ":18080/jettyserver-case/case/receiveContext-0").send();
+        Response.CompleteListener listener = result -> { };
```

Comment:
@yangyulely can you add something(such as http request) in listener callback to test it

Reply:
ok, i will update later